.pre-commit-config.yaml

```diff
@@ -33,6 +33,7 @@ repos:
     rev: v2.2.6
     hooks:
       - id: codespell
+        entry: codespell --skip="*.js,*.html,*.css,*.svg"
```

Comment:
Should we add this to the pyproject.toml config?

Reply:
yeah, I'll add that

Reply:
In theory you should be able to only declare it in `pyproject.toml` and it'll get picked up by the pre-commit invocation

Reply:
yep, that worked.